src/containers/BooksList.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+
+const BooksList = ({ books }) => {
+  const booksList = books.map(book => <Book book={book} key={Math.random()} />);
```

Comment:
- [ ] The BookList should contain the handle remove function and attached to each Book component.